training/general/src/environments/sklearn-1.1/context/README.md

```diff
@@ -0,0 +1,28 @@
+# SKLearn Docker Environment 
+
+## Overview
+This document provides details on how to create and run a sample workload in a containerized SKLearn Environment using stock version of SKLearn as well as Intel Extension for SKLearn on Azure ML instances.
+
+## How it works
+The container has stock and intel-optimized versions available. 
+To build the docker container use the `Dockerfile` and `conda_dependencies.yaml`.
+Refer to the link [here](https://github.com/Azure/azureml-assets/wiki/Environments#testing-environment-image-builds) on how the macros are replaced.
```

Comment:
I don't think this is accurate anymore, now that the extension is being installed as part of the regular image. You could include information here on how to build the image locally but it would also help to describe how to use this environment in Azure ML.

Reply:
@elesel : to build the docker image locally, we would have to replace macros in the dockerfile and conda yml file using command like `sed` which I had posted in an older version of the README.md which @rtanase had me remove it and pointed to the macros. Otherwise build will fail. 

Regarding, running on Azure ML, it is mentioned in the README, that the user would have to build the container and run `sklearn_sample_test.py` inside the container by logging in to an Azure ML instance. Do we want to add more details?

Reply:
This works. Thanks!